README.md

```diff
@@ -1,3 +1,14 @@
+# Using CLI Commands
+
+Salesforce CLI commands for custom metadata types are available in v49. The custom metadata loader is no longer supported or maintained. As such, Salesforce does not guarantee the functionality or performance of the loader. 
+
+The CLI commands simplify development and help you build automation and synchronize your source from scratch orgs when working with custom metadata types. CLI commands offer more functionality than the custom metadata loader. You can create custom metadata types, generate fields, create records, bulk insert records from a CSV file, and generate custom metadata types from an sObject. In addition, there's no limit on the number of records that can be loaded. 
+
+See the following for more information,
```

Comment:
I'm not sure of the convention but you might want to use a colon instead of a comma at the end of line here.